pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -846,12 +846,12 @@ def test_read_json_orient_table_old_schema_version(self):
         result = pd.read_json(StringIO(df_json), orient="table")
         tm.assert_frame_equal(expected, result)
 
-    @pytest.mark.parametrize("freq", ["M", "2M", "Q", "2Q"])
+    @pytest.mark.parametrize("freq", ["M", "2M", "Q", "2Q", "Y", "2Y"])
     def test_read_json_table_orient_period_depr_freq(self, freq, recwarn):
         # GH#9586
         df = DataFrame(
             {"ints": [1, 2]},
-            index=pd.PeriodIndex(["2020-01", "2020-06"], freq=freq),
+            index=pd.PeriodIndex(["2020-01", "2021-06"], freq=freq),
```

Comment:
just out of interest, why does the year need to change here?

Reply:
the reason is, when the period is less then frequency we get an assertion error
```
>       tm.assert_frame_equal(df, result)
E       AssertionError: DataFrame are different
E       
E       DataFrame shape mismatch
E       [left]:  (2, 1)
E       [right]: (2, 2)
```
I checked we have the same behavior for other frequencies, e.g.
`df = pd.DataFrame({"ints": [1, 2]}, index=pd.PeriodIndex(["2020-01-01", "2020-01-03"], freq="W"))`
raises `AssertionError: DataFrame are different`

Reply:
ah makes sense, thanks!